locales/pl/php-inline.json

```
@@ -92,10 +92,10 @@
     "password.symbols": "To pole musi zawierać przynajmniej jeden symbol.",
     "password.uncompromised": "Podane pole pojawiło się w wycieku danych. Proszę wybrać inną wartość pola.",
     "present": "Pole musi być obecne.",
-    "present_if": "This field must be present when :other is :value.",
-    "present_unless": "This field must be present unless :other is :value.",
-    "present_with": "This field must be present when :values is present.",
-    "present_with_all": "This field must be present when :values are present.",
+    "present_if": "Pole :attribute musi być obecne jeżeli :other ma wartość :value.",
```

```suggestion
    "present_if": "Podane pole musi być obecne jeżeli :other ma wartość :value.",
```
And for the remaining keys you also need to change the message format.